microsoft-edge/webview2/release-notes.md

```diff
@@ -32,6 +32,106 @@ To load WebView2, the minimum version of Microsoft Edge or the WebView2 Runtime
+
+##### [WinRT/C#](#tab/winrtcsharp)
+
+* [CoreWebView2.PostSharedBufferToScript method](/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2?view=&preserve-view=truewebview2-winrt-1.0.1466-prerelease#postsharedbuffertoscript)
+* [CoreWebView2Frame.PostSharedBufferToScript method](/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2frame?view&preserve-view=true=webview2-winrt-1.0.1466-prerelease#postsharedbuffertoscript)
+
+##### [Win32/C++](#tab/win32cpp)
+
+* [ICoreWebView2Experimental18::PostSharedBufferToScript method](/microsoft-edge/webview2/reference/win32/icorewebview2experimental18?view=webview2-1.0.1466-prerelease&preserve-view=true#postsharedbuffertoscript)
+* [ICoreWebView2ExperimentalFrame4::PostSharedBufferToScript method](/microsoft-edge/webview2/reference/win32/icorewebview2experimentalframe4?view=webview2-1.0.1466-prerelease&preserve-view=true#postsharedbuffertoscript)
+
+---
+
+*  Added support by adding these properties to run JavaScript code from the JavaScript parameter in the current top-level document:
+    *  `ColumnNumber`
+    *  `LineNumber`
+    *  `Message`
+    *  `Name`
+    *  `ToJson`
+
+##### [.NET/C#](#tab/dotnetcsharp)
+
+* [CoreWebView2ScriptException Class](/dotnet/api/microsoft.web.webview2.core.corewebview2scriptexception?view&preserve-view=true=webview2-dotnet-1.0.1466-prerelease)
+
+##### [WinRT/C#](#tab/winrtcsharp)
+
+* [CoreWebView2ScriptException Class](/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2scriptexception?view&preserve-view=true=webview2-winrt-1.0.1466-prerelease)
+
+##### [Win32/C++](#tab/win32cpp)
+
+* [ICoreWebView2ExperimentalScriptException interface](/microsoft-edge/webview2/reference/win32/icorewebview2experimentalscriptexception?view=webview2-1.0.1466-prerelease&preserve-view=true)
+
+---
+
+#### Bug fixes
+
+*   Fixed a bug which custom header title in print settings could be wrong([Issue #2093](https://github.com/MicrosoftEdge/WebView2Feedback/issues/2093))
+*   Display AllowedCertificateAuthorities in add_ClientCertificateRequested event as Base64 string. (Runtime)([Issue #2346](https://github.com/MicrosoftEdge/WebView2Feedback/issues/2346))
+*   Fixed a bug which default footer URI in print settings is missing.([Issue #2851](https://github.com/MicrosoftEdge/WebView2Feedback/issues/2851))
+*   Fixed a bug that brings null poiner exception related to print settings. (Runtime)([Issue #2858](https://github.com/MicrosoftEdge/WebView2Feedback/issues/2858))
+*   Fixed a bug that reports failure navigation when redirecting to a server that has configured with Client Certificate Authentication and when add_WebResourceRequested is subscribed. (Runtime)

```

Comment:
```suggestion
*   Fixed a bug that reports failure of navigation when redirecting to a server that has been configured with Client Certificate Authentication and when the `add_WebResourceRequested` event is subscribed to. (Runtime)
```
@peiche-jessica - which wording to use?
reports failure navigation when redirecting
reports failure of navigation when redirecting
reports navigation failure when redirecting
reports that navigation failed when redirecting

Reply:
Is there an Issue to link to?

Reply:
"reports navigation failure when redirecting"

Reply:
I edited the Suggested Change per Nishitha.